securityconfig/roles.yml

```diff
@@ -2,7 +2,7 @@ _meta:
   type: "roles"
   config_version: 2
 
-# Restrict users so they can only view visualization and dashboard on kibana
+# Restrict users so they can only view visualization and dashboard on Kibana
```

Comment:
Should this be renamed to OpenSearchDashboard?

Reply:
Done